opta/core/terraform.py

```diff
@@ -802,6 +802,7 @@ def _create_aws_state_storage(cls, providers: dict) -> None:
                     Bucket=bucket_name,
                     CreateBucketConfiguration={"LocationConstraint": region},
                 )
+            time.sleep(10)
```

Comment:
Isn’t there a better way to do this? Like polling for the buckets and existence?